packages/web-components/src/utils/utils.spec.ts

```diff
@@ -85,7 +87,7 @@ describe('getSlottedNodes()', () => {
       .spyOn(defaultElement.shadowRoot, 'querySelector')
       .mockImplementation(selector => {
         if (selector === 'slot') {
-          return mockObject;
+          return mockObject as unknown as Element;
```

Comment:
I don't like this syntax, but mockImplementation expects the return value to be of type Element and this is the way the internet suggests to do this.